spec/game_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rspec'
+require_relative '../classes/game'
+require_relative '../classes/item'
+
+describe Game do
+  let(:game_params) do
+    {
+      id: 1,
+      publish_date: '2020-01-01',
+      last_played_at: '2023-01-01',
+      multiplayer: true
+    }
+  end
+
+  subject(:game) { described_class.new(game_params) }
+
+  before do
+    allow(Date).to receive(:parse).and_call_original
+    allow(Date).to receive(:parse).with('2023-01-01').and_return(Date.new(2023, 1, 1))
+  end
+
+  describe '#initialize' do
+    it 'sets the multiplayer attribute' do
+      expect(game.multiplayer).to eq(game_params[:multiplayer])
+    end
+
+    it 'sets the last_played_at attribute' do
+      expect(game.last_played_at).to eq(Date.new(2023, 1, 1))
+    end
+  end
+
+  describe '#can_be_archived?' do
+    context 'when last played recently' do
+      it 'returns false' do
+        allow(Date).to receive(:today).and_return(Date.new(2023, 8, 1))
+        game.last_played_at = Date.new(2023, 8, 1)
+        expect(game.send(:can_be_archived?)).to be_falsey
+      end
+    end
+
+    context 'when last played more than 2 years ago' do
+      it 'returns true' do
+        allow(Date).to receive(:today).and_return(Date.new(2025, 1, 1))
+        game.last_played_at = Date.new(2023, 1, 1)
+        expect(game.send(:can_be_archived?)).to be_truthy
+      end
+    end
+  end
```

Comment:
- I think it would be better to describe this test as it is in the requirements.
> - can_be_archived?() in the Game class
>   - should override the method from the parent class
>   - should return true if parent's method returns true AND if last_played_at is older than 2 years
>   - otherwise, it should return false